### PR TITLE
Add insecure mode for wip-catalog chart

### PR DESCRIPTION
**Pull request by pmorie**
To help move things along until we have SSL stuff in place working with the aggregator.

### CODE REVIEW

**Comment by MHBauer** [comment]
Why?

**Comment by MHBauer** [comment]
I ask, as I was told not to even consider using the insecure options.

**Comment by arschles** [comment]
@MHBauer good question. This is just for development only, and will be removed as soon as we can do so and have an easy in-cluster dev/test workflow

File: cmd/apiserver/app/server/server.go [hunk 1]
```diff
@@ -205,6 +211,7 @@ func (serverOptions ServiceCatalogServerOptions) runServer() error {
 
 	// I don't like this. We're reaching in too far to call things.
 	preparedserver := server.GenericAPIServer.PrepareRun() // post api installation setup? We should have set up the api already?
+	preparedserver.SecureServingInfo = nil
```
Comment (arschles) [inline cmd/apiserver/app/server/server.go]: @pmorie I think I added this right when we started working on this PR together. I don't think it's necessary. Thoughts?

**Comment by arschles** [comment]
I restarted the stalled build in travis

**Comment by arschles** [comment]
I'm merging this now, as it's needed to unblock @pmorie and my testing efforts for crucial MVP1 functionality

**Pull request merged**